### PR TITLE
Add holidays for Sweden

### PR DESCRIPTION
**Pull request by robost**
Re-opening [https://github.com/spatie/holidays/pull/31](https://github.com/spatie/holidays/pull/31) as it was closed by mistake.

### CODE REVIEW

**Comment by robost** [comment]
Here you go @riasvdv :)

File: src/Countries/Sweden.php [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+{
+    public function countryCode(): string
+    {
+        return 'se';
+    }
+
+    protected function allHolidays(int $year): array
+    {
+        return array_merge([
+            'Nyårsdagen'       => CarbonImmutable::create($year, 1, 1),
```
Comment (Nielsvanpach) [inline src/Countries/Sweden.php:17]: Could you use the same format for fixed holidays, as in other Country classes?

**Review by Nielsvanpach — COMMENTED** [review]
Thanks! Could you look at the phpstan errors?

File: src/Countries/Sweden.php [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+            'Juldagen'         => CarbonImmutable::create($year, 12, 25),
+            'Annandag jul'     => CarbonImmutable::create($year, 12, 26),
+        ], $this->variableHolidays($year));
+    }
+
+    /** @return array<string, CarbonImmutable> */
+    protected function variableHolidays(int $year): array
+    {
+        $easter = CarbonImmutable::createFromTimestamp(easter_date($year))
+            ->setTimezone('Europe/Stockholm');
```
Comment (Nielsvanpach) [inline src/Countries/Sweden.php:30]:
```suggestion
        $easter = $this->easter($year);
```

**Comment by Nielsvanpach** [comment]
Closing as another PR for Sweden has been merged

**Pull request closed without merging**